tests/E2E/test/common.webdriverio.js

```diff
@@ -180,19 +192,19 @@ module.exports = {
       options.host = global.selenium_host;
     }
     else {
-      // We need to set it to localhost for "got" library
+      // We need to set it to localhost for 'chai-http' library
```

Comment:
You're not using chai-http :thinking:

Reply:
I replace got by chai-http in PR #13811 , like you suggested

Reply:
Cooooool :+1: